### DISCUSSION TITLE
Release

### DISCUSSION THREAD

**Discussion by cschramm** [blueman-project/blueman#1753] [General]
I'd like to discuss options for a release.

@clefebvre asks for a tag https://github.com/blueman-project/blueman/pull/1747#issuecomment-1165466266. Of course we can create an arbitrary tag just for the Mint beta but I'd also like to get 2.3 out. The changelog is by no means feature packed but we should probably release more often anyway.

There's not much in https://github.com/blueman-project/blueman/milestone/7. As I have limited time, I'm super fine with moving out the DNS stuff from the milestone (currently blocked by #1564).

#1751 bugs me. Maybe we can find a quick solution there.

I hesitate to get in #1746 as it will break translations - some of them even fixing the accelerators. I think we should postpone that.

UI wise I'd definitely like to change / replace the connection bars, maybe implement something from #1620 but I don't see me working on that anytime soon, so not considering it for a 2.3.

Any timeline @clefebvre? Any plans @infirit? :smile:

**Comment by infirit** [comment]
Fine with releasing as is. I'll look at #1751 and see how thing work for me. #1746  can be in 2.3.x.

As for the UI, I have been experimenting with Gtk.ListBox and Gtk.Revealer and that opens up how we can display things. I am not quite ready to foist my design on the world just yet but it does get rid of the custom bars we have now and replaced them with Gtk,LevelBar.

How about we put out a beta and get some feedback and i a few weeks do a final with fixes for issues that came up?
